tests/Bridge/Doctrine/Orm/Util/QueryJoinParserTest.php

```diff
@@ -43,30 +43,50 @@ public function testGetClassMetadataFromJoinAlias()
         $this->assertEquals($metadata, $classMetadata->reveal());
     }
 
+    /**
+     * @group legacy
+     * @expectedDeprecation The use of "ApiPlatform\Core\Bridge\Doctrine\Orm\Util\QueryJoinParser::getJoinRelationship()" is deprecated since 2.3 and will be removed in 3.0. Use "Doctrine\ORM\Query\Expr\Join::getJoin()" directly instead.
```

Comment:
Actually I'm not sure why we add double quotes where they're unnecessary? (And it makes the message harder to read...)

Reply:
I can remove them later, when I'll get a computer, it is just a copy/paste from an other error in the codebase somewhere though :stuck_out_tongue:

Reply:
I've checked and this is also how it is done in symfony, so I guess we should stick to this convention...?

Reply:
Oh, okay...